curriculum/challenges/english/07-scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/65687da2e60409c45595bbe1.md

```diff
@@ -20,7 +20,7 @@ You should have `if number >= 10:` within the `for` loop.
         const verify_card_number = __helpers.python.getDef("\n" + transformedCode, "verify_card_number");
         const { function_body } = verify_card_number;
 
-        assert.match(function_body, /if +number +>= +10:/);
+        assert.match(function_body, /if\s+number\s+>=\s+10:/);
```

Comment:
```suggestion
        assert.match(function_body, /if\s+number\s*>=\s*10\s*:/);
```

Reply:
As we aren't doing that, this can be resolved.